app/src/main/java/com/randomappsinc/studentpicker/home/MainActivity.java

```diff
@@ -259,6 +264,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 return true;
             case R.id.view_tutorial:
                 showTutorial(false);
+                UIUtils.hideKeyboard(this);
```

Comment:
If you update this branch against Gear61:master, this should go away. It's not part of this pull request, so let's try to get it out to keep the pull request focused. At Facebook, we have a concept of "One and only thesis", which states that every pull request should be hyper-focused and only do 1 thing.

Reply:
I think I did a mistake here, Should I merge the branch with the master and then continue the second issue?